src/open_clip/transformer.py

```diff
@@ -349,6 +472,25 @@ def forward(self, x: torch.Tensor):
 
         return x
 
+class CoCaVisionTransformer(VisionTransformer):
```

Comment:
can we make this a special case of our VisionTransformer and if there's any differences maybe we can address those in VisionTransformer nicely